src/main/splitting/tasks_tests_handler.py

```diff
@@ -0,0 +1,290 @@
+import os
+import signal
+import logging
+from subprocess import Popen, PIPE, call
+
+from src.main.util import consts
+from src.main.util.consts import LANGUAGE
+from src.main.preprocessing.activity_tracker_handler import get_extension_by_language
+from src.main.util.file_util import get_content_from_file, create_file, create_directory, remove_directory
+
+INPUT_FILE_NAME = consts.TASKS_TESTS.INPUT_FILE_NAME.value
+TASKS_TESTS_PATH = consts.TASKS_TESTS.TASKS_TESTS_PATH.value
+SOURCE_OBJECT_NAME = consts.TASKS_TESTS.SOURCE_OBJECT_NAME.value
+
+log = logging.getLogger(consts.LOGGER_NAME)
+
+
+class TimeoutException(Exception):
```

Comment:
Maybe should we move the TimeoutException class and the timeout_handler function into a separate file? Because we can reuse it in the future in other places